src/components/Camera/index.js

```diff
@@ -83,13 +85,25 @@ type CameraType = {
   ...CameraCommonType,
   onScreenshot: Function,
   trackScreen: Function,
+  onWebcamError: Function
 }
 
-export default class Camera extends React.Component<CameraType> {
+type State = {
+  cameraHeight: number | Object
+}
+
+type Props = Object
```

Comment:
I'm not sure if `Props` is doing anything, `CameraType` is supposed to be the type definition for the props